packages/core/include/combo/oot/actors/Custom_Torch2.h

```diff
@@ -0,0 +1,32 @@
+#ifndef Z_EN_TORCH2_H
+#define Z_EN_TORCH2_H
+
+#include <combo/common/actor.h>
+
+struct EnTorch2;
+
+typedef struct EnTorch2 {
+    /* 0x000 */ Actor base;
+    /* 0x13c */ ColliderCylinder collider;
+    /* 0x188 */ u8 state;
+    /* 0x189 */ u8 framesUntilNextState;
+    /* 0x18a */ s16 alpha;
+} EnTorch2; // size = 0x18c
+
+typedef enum {
+    /* 0 */ TORCH2_PARAM_HUMAN,
+    /* 1 */ TORCH2_PARAM_GORON,
+    /* 2 */ TORCH2_PARAM_ZORA,
+    /* 3 */ TORCH2_PARAM_DEKU,
+    /* 4 */ TORCH2_PARAM_FIERCE_DEITY
+} EnTorch2Param;
```

Comment:
We'd rather use defines (can be shared with ASM and doesn't create `int` aliases)